rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/RxAudioPlayer.java

```diff
@@ -78,19 +80,19 @@ public int progress() {
      * @deprecated please use {@link #play(PlayConfig)} version.
      */
     @Deprecated
-    public Single<Boolean> play(@NonNull final File audioFile) {
+    public Observable<Boolean> play(@NonNull final File audioFile) {
         return play(PlayConfig.file(audioFile).build());
     }
 
     /**
      * play audio from local file. should be scheduled in IO thread.
      */
-    public Single<Boolean> play(@NonNull final PlayConfig config) {
+    public Observable<Boolean> play(@NonNull final PlayConfig config) {
         if (config.mType == PlayConfig.TYPE_FILE && config.mAudioFile != null &&
                 config.mAudioFile.exists()) {
-            return Single.create(new Single.OnSubscribe<Boolean>() {
+            return Observable.create(new Observable.OnSubscribe<Boolean>() {
                 @Override
-                public void call(final SingleSubscriber<? super Boolean> singleSubscriber) {
+                public void call(final Subscriber<? super Boolean> singleSubscriber) {
```

Comment:
Now that it's not `SingleSubscriber` any more, we should not name it `singleSubscriber`.